TideApp/TideApp/App/TideAppApp.swift

```diff
@@ -14,7 +14,7 @@ struct TideAppApp: App {
   
   var body: some Scene {
     WindowGroup {
-      TideTimesView(viewModel: TideTimesViewModel(weatherFetcher: WeatherDataFetcher()))
+      LocationsView(viewModel: .init())
```

Comment:
super minor, May we use a more verbose init here? To me is more friendly to understand immediatly what you pass